xbmc/dialogs/GUIDialogFileBrowser.cpp

```diff
@@ -257,7 +257,7 @@ bool CGUIDialogFileBrowser::OnMessage(CGUIMessage& message)
           if (CDirectory::Create(strPath))
             Update(m_vecItems->GetPath());
           else
-            CGUIDialogOK::ShowAndGetInput(20069,20072,20073,0);
+            CGUIDialogOK::ShowAndGetInput(20069,20072);
```

Comment:
err, you accidently deleted string 20072 and 20073 is only one half of the message.